docs/scaffolding.md

```diff
@@ -1,12 +1,12 @@
 ## Scaffold generator
 
-The scaffold generator builds out the entire directory structure required for creating a particular skill.
+The scaffold generator builds out the entire directory structure required for creating an agent.
```

Comment:
From my point of view : The scaffold generator creates a new empty skill/protocol/connection etc and registers them in the aea-config.yaml. The developer must implement his custom logic. By saying that 

> builds out the entire directory structure

 it doesn't make any difference from aea create my_agent.

Reply:
"The scaffold generator builds out the directory structure required for adding new skills, protocols, and connections to the agent."